source/docs/v3/optional-features.blade.md

```diff
@@ -13,6 +13,7 @@ section: content
 - [`TenantConfig`]({{ $page->link('features/tenant-config') }}) for mapping keys from the tenant storage into the application config
 - [`CrossDomainRedirect`]({{ $page->link('features/cross-domain-redirect') }}) for adding a `domain()` macro on `RedirectResponse` letting you change the intended hostname of the generated route
 - [`UniversalRoutes`]({{ $page->link('features/universal-routes') }}) for route actions that work in both the central & tenant context
+- [`ViteBundler`]({{ $page->link('features/vite-bundler') }}) for making Vite generate the correct asset paths
```

Comment:
Add an extra break line here, the "All of the package's Features are in the `Stancl\Tenancy\Features` namespace." should be in the next line

![image](https://user-images.githubusercontent.com/13445515/193085085-6b16bff9-edd8-46fb-a33c-198eb5fbb7ce.png)

Reply:
Corrected this, thanks!